CHANGELOG.md

```diff
@@ -11,6 +11,14 @@ require manual edits to pluggable interfaces.
 
 ### Improvements to strategic projects
 
+## 1.1.1
+## Changed
+- Split a couple of functions where I was being more fancy than I needed to so I coudl conform to new typescript rules.
+## Added
+- Strategic projects and groups will now display a edit button for users that are part of the admin posix/ldap group. This button is a direct link to editing the project or group on the admin page to allow for quicker updates.
+## Security
+- Upgraded webpack-dev-server due to SVE on the 3.1.10 version.
```

Comment:
CVE*

Reply:
How did that even happen? Thanks for the sharp eyes.